projects/Mallard/src/navigation/navigators/sidebar.tsx

```diff
@@ -137,18 +135,6 @@ export const createSidebarNavigator = (
         }
     }
 
-    // -iOS12 only use Native navigator
```

Comment:
Just to check does this code has any relevance with iOS12?

Reply:
No - it was minus iOS12